qcodes/instrument_drivers/stanford_research/SR830_channels.py

```diff
@@ -0,0 +1,628 @@
+                               'AC': 0,
+                               'DC': 1,
+                           })
+
+        self.add_parameter('notch_filter',
+                           label='Notch filter',
+                           get_cmd='ILIN?',
+                           set_cmd='ILIN {}',
+                           val_mapping={
+                               'off': 0,
+                               'line in': 1,
+                               '2x line in': 2,
+                               'both': 3,
+                           })
+
+        # Gain and time constant
+        self.add_parameter(name='sensitivity',
+                           label='Sensitivity',
+                           get_cmd='SENS?',
+                           set_cmd='SENS {:d}',
+                           get_parser=self._get_sensitivity,
+                           set_parser=self._set_sensitivity
+                           )
+
+        self.add_parameter('reserve',
+                           label='Reserve',
+                           get_cmd='RMOD?',
+                           set_cmd='RMOD {}',
+                           val_mapping={
+                               'high': 0,
+                               'normal': 1,
+                               'low noise': 2,
+                           })
+
+        self.add_parameter('time_constant',
+                           label='Time constant',
+                           get_cmd='OFLT?',
+                           set_cmd='OFLT {}',
+                           unit='s',
+                           val_mapping={
+                               10e-6:  0, 30e-6:  1,
+                               100e-6: 2, 300e-6: 3,
+                               1e-3:   4, 3e-3:   5,
+                               10e-3:  6, 30e-3:  7,
+                               100e-3: 8, 300e-3: 9,
+                               1:     10, 3:     11,
+                               10:    12, 30:    13,
+                               100:   14, 300:   15,
+                               1e3:   16, 3e3:   17,
+                               10e3:  18, 30e3:  19,
+                           })
+
+        self.add_parameter('filter_slope',
+                           label='Filter slope',
+                           get_cmd='OFSL?',
+                           set_cmd='OFSL {}',
+                           unit='dB/oct',
+                           val_mapping={
+                               6: 0,
+                               12: 1,
+                               18: 2,
+                               24: 3,
+                           })
+
+        self.add_parameter('sync_filter',
+                           label='Sync filter',
+                           get_cmd='SYNC?',
+                           set_cmd='SYNC {}',
+                           val_mapping={
+                               'off': 0,
+                               'on': 1,
+                           })
+
+        def parse_offset_get(s):
```

Comment:
Normal `Parameters` should always be scalar (or strings). The `OEXP` command controls both offset and expansion (scale factor) and should be mapped to two different `Parameters`, just like on the front panel. I realise that this PR did not introduce this mistake, but it should correct it.